src/main/java/ikysil/training/ws/api/v1/dto/OrderDto.java

```diff
@@ -22,4 +22,17 @@
     private Instant createdOn;
     private Instant updatedOn;
     private Instant completedOn;
+
+    public OrderDto(String id, String orderNumber, String sourceSystem, String userId, String userEmail) {
+        this.id = id;
+        this.orderNumber = orderNumber;
+        this.sourceSystem = sourceSystem;
+        this.userId = userId;
+        this.userEmail = userEmail;
+    }
+
+    public String getId() {
```

Comment:
you already have all getters and setters, see lombok  annotation `@Data` on class.